Formula/jdupes.rb

```diff
@@ -1,21 +1,19 @@
 class Jdupes < Formula
   desc "Duplicate file finder and an enhanced fork of 'fdupes'"
   homepage "https://github.com/jbruchon/jdupes"
-  url "https://github.com/jbruchon/jdupes/archive/v1.13.2.tar.gz"
-  sha256 "41376ace274320a77e35ffed170ff0cbca2cd0cfd0ff1cea53d7464e97d4341f"
-
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "7de156d158f3a57689a6db010bc786e1becc38c594a17114e0e7992e1e541925" => :catalina
-    sha256 "5914122e74633a5d22c16102e0ffbf43c6a5d5903ecc75d41b050339af913b96" => :mojave
-    sha256 "f2132c6e2a0a1402a0dcc0205efc2ae80fcc1e70508f1910db3113de0d64ce1d" => :high_sierra
-    sha256 "95ce9083f5ee6dabdaae7ac9165b51231aa8df5240452f3eaec9c172064082cc" => :sierra
-  end
+  url "https://github.com/jbruchon/jdupes/archive/v1.14.0.tar.gz"
+  sha256 "b83285d97f1df5602647749829fdcdbcf21ece273c669bdb8e62544238b1f54e"
 
   def install
     system "make", "install", "PREFIX=#{prefix}"
   end
 
+  def caveats; <<~EOS
+    jdupes is no drop-in replacement for fdupes,
```

Comment:
This caveat doesn't look [Homebrew-specific](https://docs.brew.sh/Formula-Cookbook#caveats). You can document this in the project's README instead.

Reply:
You are right , I realised I had made mistakes in the format of my commit message and also my editing shortly after I submitted it, I should have noticed it before and thank you for pointing it out.

I am not quite sure if the warning about the incompatibilities between jdupes and fdupes should be added at all and if so, where it should be added. The caveat in the formula is not the right place indeed, and maybe the warning on jdupe's home page is enough.

Reply:
I have simplified the modifications, next time I will stick to brew bump-formula-pr for a simple version bump.